sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -719,13 +723,17 @@ public Mono<Void> openConnectionsAndInitCaches() {
             CosmosContainerProactiveInitConfig proactiveContainerInitConfig =
                 new CosmosContainerProactiveInitConfigBuilder(Collections.singletonList(cosmosContainerIdentity))
                     .setProactiveConnectionRegionsCount(1)
+                    .withMinConnectionsPerEndpointForContainer(cosmosContainerIdentity, Configs.getMinConnectionPoolSizePerEndpoint())
                     .build();
 
-            return withContext(context -> openConnectionsAndInitCachesInternal(proactiveContainerInitConfig)
-                                            .flatMap(openResult -> {
-                                                logger.info("OpenConnectionsAndInitCaches: {}", openResult);
-                                                return Mono.empty();
-                                            }));
+            return withContext(context -> openConnectionsAndInitCachesInternal(
+                    proactiveContainerInitConfig
+            )
+            .collectList() // TODO: need a way to get the open result
```

Comment:
What about this TODO - can be removed or still some pending change?

Reply:
The pending change is to log the result of several open connection tasks.